chaos-proxy/src/index.ts

```diff
@@ -15,6 +15,10 @@ const PORT = parseInt(process.env.PORT || '8545');
         throw new Error('CHAOSLABS_SIMULATIONS is not set');
     }
     const simulationId = await chooseSimulationId(CHAOSLABS_SIMULATIONS);
+    if (!simulationId) {
+        console.info('Exiting since no simulation was selected');
```

Comment:
Maybe adding a bit more context to this error message would make sense. If I ran into this error, I would be a bit lost from my perspective on how I misconfigured the simulation.

Reply:
It's not an error. It's an edge case with the library which I chose for interactive terminal prompt: when calling `control + C` instead of exiting the program it only exits the prompt and proceeds with selected answer being undefined. I haven't noticed this while first implementing it, but handled it now. You can try to run `chaos-proxy` yourself to see it:

<img width="328" alt="Screenshot 2022-04-14 at 09 19 38" src="https://user-images.githubusercontent.com/3393626/163334301-4e2114bd-e4ef-420e-86d9-b6d33b731b63.png">